Meta/gn/secondary/Userland/Libraries/LibTimeZone/BUILD.gn

```diff
@@ -32,16 +33,37 @@ if (enable_timezone_database_download) {
       "europe",
       "northamerica",
       "southamerica",
-      "zone1970.tab",
     ]
   }
+  compiled_action("generate_timezone_sources") {
+    tool = "//Meta/Lagom/Tools/CodeGenerators/LibTimeZone:GenerateTimeZoneData"
+    deps = [ ":timezone_database_files" ]
+    inputs = get_target_outputs(":timezone_database_files")
+    outputs = [
+      "$target_gen_dir/TimeZoneData.h",
+      "$target_gen_dir/TimeZoneData.cpp",
+    ]
+    args = [
+      "-h",
+      rebase_path(outputs[0], root_build_dir),
+      "-c",
+      rebase_path(outputs[1], root_build_dir),
+      "-z",
+      # NOTE: Coordinates file must be inputs[0]
+    ]
+    foreach(data_file, inputs) {
+      args += [ rebase_path(data_file, root_build_dir) ]
+    }
+  }
 }
 
 # FIXME: Another OBJECT library...
 source_set("LibTimeZone") {
   output_name = output_name_prefix + "timezone"
   include_dirs = [
     "//Userland/Libraries",
+    # FIXME: How to get the parent of the current target's gen dir?
+    "$target_gen_dir/..",
```

Comment:
What do you need this for?

I sometimes use a public_config to push an include dir to deps: https://github.com/llvm/llvm-project/blob/main/llvm/utils/gn/secondary/lld/Common/BUILD.gn#L4 (but sometimes I'm lazy and just add something to include_dirs)

Reply:
A common pattern for generated includes like this one is to #include them like so: 

```
#include <LibTimeZone/TimeZoneData.h>
```

Which assumes that we have the *parent* of the current lib's binary directory (or generated file directory) passes as an include dir. It's a bit... awkward of a pattern. I'm probably overcomplicating it though. The thing that the CMake build does is add `${CMAKE_BINARY_DIR}/Userland/Libraries` and `${CMAKE_BINARY_DIR}/Userland/Services/` to everyone's path with `include_directories()` at the top level.

Reply:
Since this is the expected pattern, I'll just remove the fixme. It could definitely be done smarter though.